test/Libraries/SystemTestServices/SystemTestServices.csproj

```diff
@@ -36,17 +36,17 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.Practices.Prism">
-      <HintPath>..\..\..\extern\prism\Microsoft.Practices.Prism.dll</HintPath>
-      <Private>False</Private>
+    <Reference Include="Microsoft.Practices.ServiceLocation, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
whats the deal with this change - the binaries changed names and there is an additional binary? I guess this change happened to all the wpf binaries?
Microsoft.Practices.Prism.dll - > Microsoft.Practices.ServiceLocation.dll?
Prism.Desktop.Net40.dll was added ?

Since the files changed names we may need to keep the old dll around for patch installers...

this whole commit seems like a great thing to get tested immediately by Civil and Revit teams.

Reply:
Good point. I totally forgot about that.

I think it may make sense to undo this and go back to using the file from `extern`. This NuGet package is the closest thing I found to the Prism we were using, but it is an unofficial package and introduces that additional dependency as you noticed.

There is a new Prism library https://github.com/PrismLibrary/Prism but it's no longer maintained by Microsoft and it is not backwards compatible. We could maybe plan to move to that in the future? For now maybe just keep using the original dll from extern.

Reply:
> Microsoft.Practices.Prism.dll - > Microsoft.Practices.ServiceLocation.dll?
Prism.Desktop.Net40.dll was added ?

`Prism.Desktop.Net40.dll` is what used to be `Microsoft.Practices.Prism.dll` and for some reason this depends on `Microsoft.Practices.ServiceLocation.dll` although the original library didn't need it.

Reply:
Let me know if you are OK with undoing this @mjkkirschner

Reply:
im good with doing the safe thing for now. We can break things for 3.0/global launch.

Reply:
Done. Tests still pass: https://master-15.jenkins.autodesk.com/view/DYN/job/DYN-DevCI_Self_Service/659/